fsw/cfe-core/src/inc/cfe_sb.h

```diff
@@ -569,10 +601,9 @@ CFE_Status_t  CFE_SB_SendMsg(CFE_MSG_Message_t *MsgPtr);
 ** \retval #CFE_SB_BAD_ARGUMENT \copybrief CFE_SB_BAD_ARGUMENT
 ** \retval #CFE_SB_MSG_TOO_BIG  \copybrief CFE_SB_MSG_TOO_BIG
 ** \retval #CFE_SB_BUF_ALOC_ERR \copybrief CFE_SB_BUF_ALOC_ERR
-**
-** \sa #CFE_SB_RcvMsg, #CFE_SB_ZeroCopySend, #CFE_SB_SendMsg
 **/
 CFE_Status_t  CFE_SB_PassMsg(CFE_MSG_Message_t *MsgPtr);
+#endif /* CFE_OMIT_DEPRECATED_6_8 */
```

Comment:
In line with my previous comment - it might be worth _not_ deprecating CFE_SB_PassMsg just yet.  I agree it should go way but I think it should be melded into the buffer-based/zero-copy approach rather than the typical transmit message approach - which should be kept as simple and straightforward as possible.